src/components/NavBar/NavigationBar.tsx

```diff
@@ -147,10 +147,10 @@ const ResponsiveAppBar: React.FC = () => {
                     </LogoElementContainer>
                     <DesktopButtonContainer>
                         {pages.map(([page, link]) => (
-                            <>
+                            <React.Fragment key={page}>
```

Comment:
ideally changes completely unrelated would be in a separate PR (with a separate title) - it makes the history really clear and random bits don't change when you do large PRs. Imagine the NavBar stopped working because of this - someone could use Git to find out it's in this PR, but then would have to review a lot of code to find out that is was here where it was changed!
I _much_ prefer lots of short PRs doing 'one semantic chunk' of things. (but not enough to make you change it - just remember this for the future!)

Reply:
Noted